blazor/rich-text-editor/style.md

```diff
@@ -7,11 +7,55 @@ control: RichTextEditor
 documentation: ug
 ---
 
-# Style and appearance in Blazor RichTextEditor Component
+# Style and Appearance
 
-The following content provides the exact CSS structure that can be used to modify the control's appearance based on the user preference.
+## Set placeholder
 
-## Customizing the Rich Text Editor's content
+The [Placeholder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.SfRichTextEditor.html#Syncfusion_Blazor_RichTextEditor_SfRichTextEditor_Placeholder) property is used to create a placeholder for the Rich Text Editor's content when the editor body is empty. 
+
+Use the `e-rte-placeholder` class to define the custom font family, font color, and styles to the placeholder text.
+
+```css
+
+.e-richtexteditor .e-rte-placeholder {
+    font-family: monospace;
+}
+
+```
+
+{% tabs %}
+{% highlight razor %}
+
+@using Syncfusion.Blazor.RichTextEditor
+
+<SfRichTextEditor Placeholder="Type something" />
+
+<style>
+    .e-richtexteditor .e-rte-placeholder {
+        font-family: monospace;
+    }
+</style>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor RichTextEditor with placeholder](./images/blazor-richtexteditor-placeholder.png)
+
+## Source code view 
+
+The Rich Text Editor allows users to directly edit HTML code via `Source View` in the text area. If you make any direct modifications in the source view, the changes will be reflected in the editor content. So, the users will have more flexibility over the content they have created.
```

Comment:
The Rich Text Editor allows you to directly edit HTML code via `Source View` in the text area. If you make any direct modifications in the source view, the changes will be reflected in the editor content. So, you will have more flexibility over the content they have created.